platform/security/interceptor/security-interceptor-guest/src/main/java/org/codice/ddf/security/interceptor/GuestInterceptor.java

```diff
@@ -81,7 +80,12 @@
 
     private boolean guestAccessDenied = false;
 
-    private final Map<String, Subject> cachedGuestSubjectMap = new ConcurrentHashMap<>();
+    private static final Cache<String, Subject> CACHED_GUEST_SUBJECT = CacheBuilder.newBuilder()
+            .expireAfterWrite(1, TimeUnit.DAYS)
+            .maximumSize(1000)
+            .build();
+
+    Security security = Security.getInstance();
```

Comment:
You should be able to inject this in the blueprint instead of calling `getInstance()`. Makes unit testing easier :-)